Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -419,6 +419,7 @@ def audit_formula_name
 
     @@local_official_taps_name_map ||= Tap.select(&:official?).flat_map(&:formula_names)
                                           .each_with_object({}) do |tap_formula_full_name, name_map|
+      next if tap_formula_full_name.start_with?("homebrew/science/")
```

Comment:
Arguably we could just kill these checks entirely given we're wanting to move more and more stuff from taps to core. Thoughts?

Reply:
Yeah. If anything, duplicates in non-Homebrew taps would be more pertinent information at this point.